docs/src/windows.md

```diff
@@ -13,6 +11,7 @@ git clone https://github.com/Qwant/erdapfel.git
 ```
 
 - Open a Ubuntu bash in this folder (shift + right-click > Open Linux CLI here)
+- Install [nvm](https://github.com/nvm-sh/nvm#install--update-script), then node 12 (`nvm install 12` + `nvm use 12`), then update npm (`npm install -g npm@latest`)
```

Comment:
For now we rely on npm 7, so it may be safer to specify `npm@7` instead of `npm@latest`.